conf/st2.package.conf

```diff
@@ -52,7 +52,7 @@ api_url =
 base_path = /opt/stackstorm
 
 [syslog]
-host = localhost
+host = 127.0.0.1
```

Comment:
rabbitmq I cheked by default is also there. So it makes sence to check also: mongo, postgres and rsyslog. Then it will be clear. Overall +1 for these changes.